--- libklvanc-vid.obe.1.6.0/configure.ac.orig	2019-11-13 22:38:45.000000000 +0100
+++ libklvanc-vid.obe.1.6.0/configure.ac	2019-11-20 21:07:08.114284576 +0100
@@ -13,8 +13,8 @@ AC_PROG_CC
 AC_PROG_CC_C99
 if test `uname -s` = "Darwin"
 then
-CFLAGS+=" -I/usr/local/include "
-LDFLAGS+=" -L/usr/local/lib "
+CFLAGS="$CFLAGS -I/usr/local/include "
+LDFLAGS="$LDFLAGS -L/usr/local/lib "
 fi
 
 # We always build with -fPIC in case our static libraries end up
